Create ui-c8y-1021-59-2-fixing-issue-on-redirect-to-last-tab-on-dashboards.md (#3182)
Cumulocity-IoT/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-59-2-fixing-issue-on-redirect-to-last-tab-on-dashboards.md

---
date: ""

> title: Fixed issue with redirecting to the last viewed tab on dashboards

Reviewer: ```suggestion
title: Redirecting to the last viewed tab on a dashboard now works properly
```
